File: curso_poo_uber/PHP/UberVan.php
<?php
require_once('CarAdvanced.php');
class UberVan extends CarAdvanced{
    public function __construct($license, $driver, $typeCarAccepted, $seatsMaterial) {
        parent::__construct($license, $driver, $typeCarAccepted, $seatsMaterial);
    }
}
?>

File: curso_poo_uber/java/UberVan.java
package platzi.curso_poo_uber.java;

//import java.util.*;

public class UberVan extends CarAdvanced{

    public UberVan(String licence, Account driver) {
        super(licence, driver);
    }
    
    @Override
    public void setPassenger(Integer passenger) {
        super.setPassenger(passenger);
    }
}
